crates/router/src/types/api/webhooks.rs

```diff
@@ -73,10 +76,71 @@ pub trait IncomingWebhook: ConnectorCommon + Sync {
 
     async fn get_webhook_source_verification_merchant_secret(
         &self,
-        _db: &dyn StorageInterface,
-        _merchant_id: &str,
+        db: &dyn StorageInterface,
+        merchant_id: &str,
+        connector_label: &str,
     ) -> CustomResult<Vec<u8>, errors::ConnectorError> {
-        Ok(Vec::new())
+        let debug_suffix = format!(
+            "For merchant_id: {}, and connector_label: {}",
+            merchant_id, connector_label
+        );
```

Comment:
instead of passing values like these, you can add fields to the log statement so that it can be parsed using the `json` formatter in grafana.

```rust
logger::warn!("Error occured",connector_label)
```
 I think we are already recording the `merchant_id` in logs, but for `connector_label` you might have to look into the [formatter.rs](https://github.com/juspay/hyperswitch/blob/b1ed93468cf8c54f2ae53420c0293a2e5a15fca4/crates/router_env/src/logger/formatter.rs) file to add this key in the extras parameters if its not being displayed